.gitignore

```diff
@@ -1,14 +1,44 @@
 # OS et IDE files
 .DS_Store
 .idea/
-.vscode/
```

Comment:
**question:** Why removing IDE specific files?

Reply:
This is due to replacing the root `.gitignore` file with the one in the template, I will fix it.